packages/paste-website/src/pages/components/separator/index.mdx

```diff
@@ -0,0 +1,151 @@
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/separator/"}}) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Separator"
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/master/packages/paste-core/components/separator"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/components-separator--horizontal"
+  data={props.data.allPasteComponent.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<TableOfContents headings={props.data.mdx.headings} />
+
+<content>
+
+## Guidelines
+
+### About Separator
+
+<Paragraph>{props.pageContext.frontmatter.description}</Paragraph>
+
+It can be used to create separation between large sections of page content, or between smaller parts of single UI controls such as menu items in a menu.
+
+## Examples
+
+### Vertical separation
+
+Vertical separation is achieved by setting the `orientation` property to horizontal. A `verticalSpacing` styling property is provided to enable you to create space above and below the separator. `verticalSpacing` will take any [Spacing Token](/tokens#spacings) as it's value.
+
+<LivePreview scope={{Separator}} language="jsx">
+  {`<Separator orientation="horizontal" verticalSpacing="space50" />`}
+</LivePreview>
+
+### Horizontal separation
+
+<LivePreview scope={{Separator, Text, Box}} language="jsx">
+  {`<Box display="flex">
+  <Text as="span">Option 1</Text>
+  <Separator orientation="vertical" horizontalSpacing="space40" />
+  <Text as="span">Option 2</Text>
+  <Separator orientation="vertical" horizontalSpacing="space40" />
+  <Text as="span">Option 3</Text>
+  <Separator orientation="vertical" horizontalSpacing="space40" />
+  <Text as="span">Option 4</Text>
+  <Separator orientation="vertical" horizontalSpacing="space40" />
+  <Text as="span">Option 5</Text>
+</Box>`}
+</LivePreview>
+
+### Spacing
+
+The Separator comes with 2 spacing properties; `horizontalSpacing` and `verticalSpacing`. You can use these properties to create space between the Separator and the two pieces of UI it sits between.
+
+When used the spacing properties create equal space in both directions along the orientation you choose.
+
+<LivePreview scope={{Separator, Text, Box}} language="jsx">
+  {`<>
+  <Box display="flex">
+    <Text as="span">Option 1</Text>
+    <Separator orientation="vertical" horizontalSpacing="space180" />
+    <Text as="span">Option 2</Text>
+  </Box>
+  <Separator orientation="horizontal" verticalSpacing="space180" />
+  <Box display="flex">
+    <Text as="span">Option 1</Text>
+    <Separator orientation="vertical" horizontalSpacing="space80" />
+    <Text as="span">Option 2</Text>
+  </Box>
+</>`}
+</LivePreview>
+
+## When to use the Separator
+
+A Separator can be used to create visual separation between two parts of a page or component composition. It can also be used as a means of creating heirarchy within a page.
```

Comment:
- "heirarchy" --> "hierarchy"
- "used as a means of creating heirarchy within a page" --> "used to create hierarchy within a page"